.gitignore

```diff
@@ -98,3 +98,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+#Files i create just for debugging
+debugbug.py
```

Comment:
@EltayebAhmed The global .gitignore should not be used for masking personal files. There is a `.git/info/exclude` file that can be used for that.